### PR TITLE
fix: Make quoted string replacement work.

### PR DESCRIPTION
**Pull request by ap891843**
Signed-off-by: ap891843 <aman.prashant@broadcom.com>

### CODE REVIEW

File: server/src/test/java/org/eclipse/lsp/cobol/usecases/TestCopybookReplacePatterns.java [hunk 1]
```diff
@@ -109,6 +115,9 @@
           + "           03  {$*LDAY^DMAN1230000000000000000000000000000000000000000000001111111111111111111111111111111111111111111100000000000000000000000}           PIC S9(7) COMP-3.";
   private static final String REPL7_NAME = "REPL7";
 
+  private static final String REPL8 = "0      01 {$*'XXX_ID$'^ACC_ID}    PIC 9.\n";
```
Comment (temanbrcom) [inline server/src/test/java/org/eclipse/lsp/cobol/usecases/TestCopybookReplacePatterns.java:118]: Please, add also a check for `01 {$*'XXX'-GROUP-NAME^ABC-GROUP-NAME}.` to verify the #1211.

**Review by grianbrcom — APPROVED** [review]
:eagle:

**Review by temanbrcom — APPROVED** [review]
🐝

**Review by grianbrcom — APPROVED** [review]
:surfer:

**Pull request merged**